### PR TITLE
Created responsive navbar

### PR DESCRIPTION
**Pull request by rajatgupta24**

### CODE REVIEW

File: css/home.css [hunk 1]
```diff
@@ -19,6 +19,21 @@ nav{
...
+    transition: transform .2s ease-in;
+}
+
+.line2, .line3, .line1{
+    margin: 4px 0;
+    width: 25px;
+    height: 3px;
+    border-radius: 3px;
+    background: black;
+}
```
Comment (tusharojha) [inline css/home.css]: set the color to white

File: index.html [hunk 2]
```diff
@@ -13,14 +13,17 @@
...
-                    <li><a href="#">Experience</a></li>
-                    <li><a href="#">Contact</a></li>
-                </ul>
+            <ul class="nav-list">
+                <li><a href="#">Home</a></li>
+                <li><a href="#">Skills</a></li>
+                <li><a href="#">Education</a></li>
+                <li><a href="#">Experience</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
```
Comment (tusharojha) [inline index.html]: Change the order to Home Education Skills Experience Contact

File: js/home.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+console.log("rajat logged");
+
+const hamburger = document.querySelector(".square");
+
+const nav = document.querySelector(".nav-list");
+
+hamburger.addEventListener("click", () => {
+    nav.classList.toggle("slide");
+});
```
Comment (tusharojha) [inline js/home.js]: add some on click animation to hamburger icon as well.
example rotate and change to X icon.

**Review by tusharojha — CHANGES_REQUESTED** [review]
Please check why there is a horizontal scrollbar?
For a responsive webpage, we target to remove the horizontal scroll bar.

File: index.html (hunk 2 above)
Comment (rajatgupta24) [reply index.html]: Done

File: css/home.css (hunk 1 above)
Comment (rajatgupta24) [reply css/home.css]: Okay

**Comment by rajatgupta24** [comment]
I think it looks good with a plain background, any ideas😄.

educat

File: index.html (hunk 2 above)
Comment (tusharojha) [reply index.html]: Change the order in which they are shown.
I don't think it is done yet.
<li><a href="#">Home</a></li>
                <li><a href="#">Education</a></li>
                <li><a href="#">Skills</a></li>
                <li><a href="#">Experience</a></li>
                <li><a href="#">Achievements</a></li>
                <li><a href="#">Contact</a></li>

File: js/home.js (hunk 3 above)
Comment (tusharojha) [reply js/home.js]: This is not the priority but still, if you have time then please add this as well.

**Comment by tusharojha** [comment]
can you add connected bullet points on the left side of the headings of the education section so that it will look like a timeline?

**Comment by rajatgupta24** [comment]
This looks good to me but on scroll it overlaps the nav & i'm not able to fix this can you tell how can i do this.

**Comment by tusharojha** [comment]
Yes @rajatgupta24 
It looks really nice and you have done a great job!

Regarding the problem of section overlapping the navbar, you can solve it using the css-property called z-index.

for example in the css styles of nav you can set its value to 1.

```css
nav {
 z-index: 100;
}
```

**Comment by tusharojha** [comment]
Also, I have a few suggestions for the achievements section.
- Convert the achievements in a card layout with its heading, date & description in order
- Use white background & black shades for fonts and try the `font-family: 'Ubuntu', sans-serif !important;`

**Comment by tusharojha** [comment]
@rajatgupta24 last thing for cards is that rather than showing a border, I think a box-shadow effect of black will look good. WDYT?
After that we are good to merge.

**Comment by tusharojha** [comment]
@rajatgupta24 please make the + icon to X by rotating 45deg

**Comment by rajatgupta24** [comment]
Done😀

**Comment by tusharojha** [comment]
Thanks

**Pull request merged**